gr-filter/python/filter/design/filter_design.py

```diff
@@ -26,19 +26,17 @@
     raise SystemExit('Please install NumPy to run this script (https://www.np.org/)')
 
 try:
-    from numpy.fft import fftpack as fft_detail
+    import numpy.fft as fft_detail
```

Comment:
Maybe also rename `fft_detail` to `fft` throughout the code and change this to `from numpy import fft`.